aspnet/testing/integration-testing.rst

```diff
@@ -66,7 +66,7 @@ Note that we're not really trying to test the correctness of our prime number ch
 
 .. image:: integration-testing/_static/test-explorer.png
 
-.. note:: You can learn more about the unit tests in the :doc:`unit-testing` article.
+.. note:: You can learn more about the unit tests in the `Unit testing <https://docs.microsoft.com/en-us/dotnet/articles/core/testing/unit-testing-with-dotnet-test>`_ article.
```

Comment:
Mine would not build locally until I put .. _Unit testing: .. https://docs.microsoft.com/en-us/dotnet/articles/core/testing/unit-testing-with-dotnet-test in the integration testing doc.